packages/carbon-web-components/src/components/code-snippet/code-snippet.ts

```diff
@@ -365,9 +383,9 @@ class CDSCodeSnippet extends FocusMixin(LitElement) {
       // Ensures no extra whitespace text node
       // prettier-ignore
       return html`
-        <cds-copy button-class-name="${classes}" @click="${handleCopyClick}">
-          <code slot="icon"><slot></slot></code>
-          <span slot="tooltip-content">${tooltipContent}</span>
+        <cds-copy button-class-name="${classes}" @click="${handleCopyClick}"  part="inline-copy">
+          <code slot="icon" part="content content-inline"><slot></slot></code>
+          <span slot="tooltip inline-tooltip" part="tooltip">${tooltipContent}</span>
```

Comment:
The `slot` attribute value should not change. This is causing a regression where the tooltip shows up as empty in the Code Snippet > Inline story.

```suggestion
          <span slot="tooltip-content" part="tooltip tooltip--inline">${tooltipContent}</span>
```